man/mysqld_safe_helper.1

```diff
@@ -9,7 +9,7 @@
 .\" disable justification (adjust text to left margin only)
 .ad l
 .SH NAME
-mysqld_safe_helper \- helper script
+mariadbd-safe-helper \- helper script (mysqld_safe is now a symlink to mariadbd-safe) (mysqld_safe_helper is now a symlink to mariadbd-safe-helper)
```

Comment:
This seems to be a copy-and-paste error

Reply:
Indeed. I had them fixed locally but forgot to stage it before commit&push. Thank you for pointing it out!